backend/bot/index.js

```diff
@@ -57,12 +62,12 @@ const postMatchResult = async (match) => {
   const winningPlayers = winningTeam.map(player => `${player.name} (${player.rating})`)
   const losingPlayers = losingTeam.map(player => `${player.name} (${player.rating})`)
 
-  const isComedyDuo = winningPlayers.length == 2 && winningTeam.every(player => player.name === 'Pepa' || player.name === 'Tonda')
-
   const messageParts = []
 
-  if (isComedyDuo) {
-    messageParts.push(':tondab: :pepadab:')
+  const { prefix, suffix } = winningTeam.length === 2 ? getMatchResultPrefixSuffix(winningTeam) : {}
```

Comment:
why is it limited to 2 people?

Reply:
Frankly, I didn't have the time to deal with that option. It can be added later.